tests/Sitecore.AspNetCore.SDK.RenderingEngine.Tests/TagHelpers/Fields/ImageTagHelperFixture.cs

```diff
@@ -739,6 +739,456 @@ public void Process_RenderingChromesAreNotNull_ChromesAreOutput(
+        sut.Sizes = "(min-width: 768px) 600px, 300px";
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        string content = tagHelperOutput.Content.GetContent();
+        content.Should().Contain("srcset=");
+        content.Should().Contain("600w");
+        content.Should().Contain("300w");
+        content.Should().Contain("sizes=\"(min-width: 768px) 600px, 300px\"");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithJsonString_GeneratesSrcSetAttribute(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        tagHelperOutput.TagName = "img";
+        sut.For = GetModelExpression(new ImageField(_image));
+        sut.SrcSet = "[{\"mw\": 500}, {\"mw\": 250}]";
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        tagHelperOutput.Attributes.Should().ContainSingle(a => a.Name == "srcset");
+        string srcSetValue = tagHelperOutput.Attributes["srcset"].Value.ToString()!;
+        srcSetValue.Should().Contain("500w");
+        srcSetValue.Should().Contain("250w");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithMixedParameterTypes_GeneratesSrcSetAttribute(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        tagHelperOutput.TagName = RenderingEngineConstants.SitecoreTagHelpers.ImageHtmlTag;
+        sut.For = GetModelExpression(new ImageField(_image));
+        sut.SrcSet = new object[]
+        {
+            // Only w and mw are supported by Content SDK for srcSet
+            new { w = 800 },  // Anonymous object with 'w'
+            new { mw = 400 }, // Anonymous object with 'mw'
+        };
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        string content = tagHelperOutput.Content.GetContent();
+        content.Should().Contain("srcset=");
+        content.Should().Contain("800w");
+        content.Should().Contain("400w");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithInvalidJsonString_FallsBackGracefully(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        tagHelperOutput.TagName = "img";
+        sut.For = GetModelExpression(new ImageField(_image));
+        sut.SrcSet = "invalid json string";
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        // Should not throw and should not add srcset attribute
+        tagHelperOutput.Attributes.Should().NotContain(a => a.Name == "srcset");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithImageParamsConflict_SrcSetParametersWin(
```

Comment:
Lets rename this to `Process_SrcSetWithImageParamsConflict_SrcSetParametersArePreferred`